compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.h

```diff
@@ -44,6 +44,7 @@ struct GPUPipelineOptions {
   bool enableReduceSharedMemoryBankConflicts = true;
   bool prefetchSharedMemory = false;
   bool enableUkernels = false;
+  bool generateContract = true;
```

Comment:
I think changes to these options need to be reflected on the tablegen side (in the matching attribute)? @Max191